ssh/server.go

```diff
@@ -60,6 +60,7 @@ func (s *Server) sessionHandler(session sshserver.Session) {
 	}).Info("Handling session request")
 
 	sess, err := NewSession(session.User(), session)
+	sess.finish() // nolint:errcheck
```

Comment:
I don't understand what this fixes. This is finishing it prior logging the error.